src/site/content/en/fast/lazy-loading/index.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: Lazy-loading
```

Comment:
Would be good to have a title that matches the pattern of the rest of the content on /fast. Maybe "Use lazy-loading to improve loading speed"?